### DISCUSSION TITLE
2.1.1

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#11057] [General]
*  Disable DNS tunneling by default while an issue with docker is debugged
*  Enable implicit ICTIMESYNCFLAG_SYNC patch (solves #10006)
*  Update MSRDC to 1.2.5105
*  Update Microsoft.WSL.Kernel to version 5.15.146.1-2
*  Update WSLg to 1.0.60
*  Ignore localhostForwarding setting in mirrored mode

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.1.1'>2.1.1</a>.</em>

**Comment by erikboto** [comment]
Is there some way to get the sources for the kernel for this prerelease? I use this to enable some mode kernel modules I need, but want to base it on the correct kernel version to also get the fix for the clock skew issue. I couldn't find any tag/branch matching this version at https://github.com/microsoft/WSL2-Linux-Kernel

**Reply by kelsey-steele** [reply]
Hi @erikboto 

The 5.15.146.1 WSL2 kernel release is now available: https://github.com/microsoft/WSL2-Linux-Kernel/releases/tag/linux-msft-wsl-5.15.146.1

**Reply by erikboto** [reply]
Awesome, thanks!

**Comment by dennismuchene** [comment]
www.google.com

**Comment by muratyurdakul75** [comment]
Hi

There are 2 NUMA on servers or some computers. Is it planned to bring support for 2 NUMA? It's currently using only 1 of 2 CPUs.

**Reply by dennismuchene** [reply]
Hi there,

I hope you are great, please let me try working and if the work is good or great. I am happy to have worked.

Sincerly,
Dennis Muchene

Cardiff,
Wales,
The United Kingdom

County Hall
Cardiff Bay
CF10 4UW
Wales
